convex_test.go

```diff
@@ -13,12 +13,12 @@ import (
 )
 
 func clear(c *ConvexDB) {
-	c.mutation(&UdfExecution{Path: "clear", Args: []interface{}{}})
+	c.mutation(&UdfExecution{Path: "clear", Args: map[string]interface{}{}, Format: "json"})
 }
 
 // Test saving and loading links for SQLiteDB
 func Test_Convex_SaveLoadLinks(t *testing.T) {
-	db := NewConvexDB("https://feeble-gull-946.convex.cloud", "test")
```

Comment:
should these changes be committed? not sure if it really matters

Reply:
Yeah - this is just my dev deployment instead of yours
Let me see if I can have it read from .env.local

Reply:
got it to work!